### ISSUE TITLE
Can we optionally output the tsq file during timestamping

### ISSUE THREAD

**Issue by rapidclock** [trbs/rfc3161ng#22]
It would be nice to get the tsq file as an output along with the tsr (response) from the timestamp function.

I know we can verify the timestamp using only the data and the tsr, but alternatively we can also verify the timestamp using tsr and tsq. (eg: `openssl ts -verify -in <tsr_file> -queryfile <tsq_file> -CAfile cert_file`)

For this purpose, it would be nice to get the tsq back from the `timestamp` fuction.

Alternatively, it would also be nice to perform the timestamping with a pre-existing tsq (that is created by the `make_timestamp_request` method)

**Comment by trbs** [comment]
yeah, in general I would love to make/change the API to be more flexible.
(Not loving things like `return_tsr` options as it is right now as well)

Regretfully I do not have time to work on this myself at the nor a sponsor to sponsor the work.

But I will leave this open with `enhancement` label, if somebody wants to make a PR or I get some time or a sponsor we can pick this up! :-)
